### PR TITLE
🤖 call actionlint Docker image directly

### PR DESCRIPTION
**Pull request by kevinmatthes**
This PR reduces the maintenance effort for the GHA workflow linter by calling its Docker image directly with the version being set to `latest` instead of applying the wrapper around it.

@tobealive, if you prefer to install updates for the linter by merging PRs, we might need to switch to Renovate as this dependency type is not recognised by Dependabot, as far as I know.

### CODE REVIEW

**Review by tobealive — APPROVED** [review]

**Comment by tobealive** [comment]
Thank you @kevinmatthes,

what is your experience otherwise with renovate? You continued using it?

**Comment by kevinmatthes** [comment]
@tobealive

At the moment, I am trying to migrate from Dependabot to Renovate with the repository of Aeruginous (https://github.com/kevinmatthes/aeruginous-rs; a CLI with numerous tools for repository maintenance) and it works very well; in the long term, I would like to switch to Renovate entirely with each of my repositories.

The basic Dependabot functionalities were very to easy to configure, now I am waiting for the Clap project to publish a new release to see if also the advanced configurations work as expected.  The main advantages are the self-merging of its own suggestions (does not work with Aeruginous due to mandatory CODEOWNER reviews but I saw it working in other repositories), entire control to the commit message (you can easily hard-code "⬆️ update" as PR and commit prefix), as well as the GitHub release signalling (you can have Renovate listen to a GitHub repository for new releases and update certain files in your repositories using regexes, then; this is why I wait for Clap updates).

If you would like to give it a try, you will need to install the free GitHub App for at least this repository.  Renovate will suggest you an initial configuration in a new PR, then.  I would adjust this configuration to meet this project's requirements and, as soon as you merged the setup PR, Renovate will begin to suggest you updates.  We can leave Dependabot configured such that we can always switch back to it.

**Pull request merged**
